UpcomingMovies.Core/AppStart.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Threading.Tasks;
+using MvvmCross.Navigation;
+using MvvmCross.ViewModels;
```

Comment:
Did you have experience with other architectural frameworks with Xamarin? How about other IoC containers? How do you like MvvmCross comparing to others?

Reply:
I played around a bit with MvvmLight but that's about it. I chose MvvmCross because that's what we use where I work.
I did read up about other frameworks like Prism and FreshMvvm but what I like about MvvmCross is that they have binding for native components, so I could re-write the project with native UI if needed without changing the core project.